core-bundle/src/Image/PictureFactory.php

```diff
@@ -164,6 +165,12 @@ private function createConfig($size): array
 
                 if (null !== $imageSizes) {
                     $options->setSkipIfDimensionsMatch((bool) $imageSizes->skipIfDimensionsMatch);
+
+                    $config->setFormats(
```

Comment:
Are you sure this works? Afaics the values are encoded in the db? Also, we need the same option in the new contao.image.sizes altough there I guess we should accept an array notation?

Reply:
Yes, works :)

Added formats to `contao.image.sizes` in 7465bcc30bad971b0a975a1279d813d2d8c2d3c4